example/storybook-nativewind/src/hooks/useMedia/index.ts

```diff
@@ -0,0 +1,32 @@
+import { useState, useEffect } from 'react';
+import { useWindowDimensions } from 'react-native';
+import { getTailwindConfigBreakpoints } from '@/utils/getTailwindConfigBreakpoints';
+
+const breakpoints = getTailwindConfigBreakpoints();
+
+const useMedia = () => {
+  const { width } = useWindowDimensions();
+  const [media, setMedia] = useState({
+    'base': false,
+    'sm': false,
+    'md': false,
+    'lg': false,
+    'xl': false,
+    '2xl': false,
+  });
+
+  useEffect(() => {
+    setMedia({
```

Comment:
remove hardcode values of breakpoints.

Reply:
I haven't added any hardcode values those were coming bydefault from `tailwind`